spec/generators/rails-template.rb

```diff
@@ -0,0 +1,269 @@
+# Generate a Rails app.
+# Use the rake tasks in this repository to generate the Rails apps.
+
+def add_gems
+  gem 'sorbet-rails', path: '../../../.'
+
+  sorbet_version = ENV["SORBET_VERSION"]
+  if sorbet_version
+    # mostly used to test against a stable version of Sorbet in Travis.
+    gem 'sorbet', sorbet_version
+    gem 'sorbet-runtime', sorbet_version
+  else
+    # prefer to test against latest version because sorbet is updated frequently
+    gem 'sorbet'
+    gem 'sorbet-runtime'
+  end
+end
+
+def add_routes
+  route "get 'test/index' => 'test#index'"
+end
+
+def create_initializers
+  initializer "sorbet_rails.rb", <<~RUBY
+    # typed: strict
+    require(Rails.root.join('lib/mythical_rbi_plugin'))
+    SorbetRails::ModelRbiFormatter.register_plugin(MythicalRbiPlugin)
+  RUBY
+end
+
+def create_lib
+  lib "mythical_rbi_plugin.rb", <<~'RUBY'
```

Comment:
```suggestion
  lib "mythical_rbi_plugin.rb", <<~RUBY
```

Reply:
This is necessary because otherwise the `#{}` will get interpreted here, instead of being inserted into the file.

Reply:
Oh, that's interesting. Thanks for clarification.